docusaurus.config.js

```diff
@@ -141,7 +142,7 @@ const config = {
       algolia: {
        appId: 'SPPRT3GDNI',
        apiKey: 'e02fb8e0c4d8c7968396981d7ecb9fa8',
-       indexName: 'dev_ngrok',
+       indexName: (process.env.DEPLOY_ENV || 'dev') + '_ngrok',
```

Comment:
should this be `|| 'prod'` since you pass `dev` in the github workflow? I assume prod doesn't export a `DEPLOY_ENV` variable

Reply:
Nevermind. I see the `DEPLOY_ENV=prod` at the top! 👍